amanuensis/resources/project/__init__.py

```diff
@@ -144,9 +144,12 @@ def update(project_id, approved_url, filter_set_ids):
         return update_project(session, project_id, approved_url)
 
 
-def upload_file(bucket, key, project_id, expires=None):
+def upload_file(key, project_id, expires=None):
+
+    bucket = config["S3_BUCKETS"]["DATA_DELIVERY_S3_BUCKET"]
```

Comment:
@paulmurdoch19 can you check in the terraform where we create this if we push the name in any config or env variable?

Reply:
ok, it is pushed in the cred.json under amanuensis.data_delivery_bucket

Reply:
ok I can change that real quick in amanuensis

Reply:
oh, I remember. The name of the S3 bucket is the key in config["AWS_CREDENTIALS"]. The string `DATA_DELIVERY_S3_BUCKET` is replaced with the actual bucket name. We only have this one bucket in amanuensis so for now it would probably be fine to just get the first and only item in the config["AWS_CREDENTIALS"] list.

However a better solution would be to keep the string `DATA_DELIVERY_S3_BUCKET` in the `AWS_CREDENTIALS` section and add a section in the S3_BUCKETS like:

```
the_buxket_name:
    cred: 'DATA_DELIVERY_S3_BUCKET'
    region: 'us-east-1'
```
This would need to be added in the cloud-auto repo. This is where the AWS_CREDENTIALS portion is built: https://github.com/chicagopcdc/cloud-automation/blob/0a04aa810cdcb10a12081a998014f2a85d30b6f0/apis_configs/config_helper.py#L130-L144

Reply:
Yes I agree this is how you would have to get the name out from the config right now list(config['AWS_CREDENTIALS'].keys())[0]

Reply:
ok the quick fix has been added to amanuensis